apps/s_time.c

```diff
@@ -59,17 +59,20 @@ static const size_t fmt_http_get_cmd_size = sizeof(fmt_http_get_cmd) - 2;
 
 typedef enum OPTION_choice {
     OPT_ERR = -1, OPT_EOF = 0, OPT_HELP,
-    OPT_CONNECT, OPT_CIPHER, OPT_CERT, OPT_NAMEOPT, OPT_KEY, OPT_CAPATH,
-    OPT_CAFILE, OPT_NOCAPATH, OPT_NOCAFILE, OPT_NEW, OPT_REUSE, OPT_BUGS,
-    OPT_VERIFY, OPT_TIME, OPT_SSL3,
+    OPT_CONNECT, OPT_CIPHER, OPT_CIPHERSUITES, OPT_CERT, OPT_NAMEOPT, OPT_KEY,
+    OPT_CAPATH, OPT_CAFILE, OPT_NOCAPATH, OPT_NOCAFILE, OPT_NEW, OPT_REUSE,
+    OPT_BUGS, OPT_VERIFY, OPT_TIME, OPT_SSL3,
     OPT_WWW
 } OPTION_CHOICE;
 
 const OPTIONS s_time_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
     {"connect", OPT_CONNECT, 's',
      "Where to connect as post:port (default is " SSL_CONNECT_NAME ")"},
-    {"cipher", OPT_CIPHER, 's', "Cipher to use, see 'openssl ciphers'"},
+    {"cipher", OPT_CIPHER, 's',
+     "TLSv1.3 and below cipher list to use, see 'openssl ciphers'"},
```

Comment:
This description is inconsistent with the `-cipher` option in apps/apps.h; which says "TLSv1.2 and below..."

Reply:
I dislike a reference to a manpage, if that's even what it is.  That should be implicit in every command; see the manpage for full(er) information.  Yes, I probably did this text.  But what does the single-quoted text *mean*?

Reply:
> This description is inconsistent with the -cipher option in apps/apps.h; which says "Specify TLSv1.2 and below..."

It should say TLSv1.2 and below.

> I dislike a reference to a manpage, if that's even what it is. That should be implicit in every command; see the manpage for full(er) information. Yes, I probably did this text. But what does the single-quoted text mean?

I think it refers to the output from running that command - but its not that clear. I can remove that bit of the text.

Reply:
Done.